docs/wiki/EL/el-specs.md

```diff
@@ -0,0 +1,1335 @@
+| 4 & 5      | 36           | $$ TRIE(L_s(\sigma)) = P(B_H)_{H_{stateRoot}} $$          | The equation defines the Parent block's state root header as the root given by the TRIE function where $P(B_H)$ is the Parent Block                                                                                                                                                                                                                                                                              |
+| 6.         | 33b          | $$ H_{stateRoot} \equiv TRIE(L_s(\Pi(\sigma, B))) $$      | this gives us the state root of the current block                                                                                                                                                                                                                                                                                                                                                                |
+
+The specified procedure for the state transition function in the code documentation includes the following steps:
+
+1. **Retrieve the Header**: Obtain the header of the most recent block added to the chain, referred to as the parent block.
+2. **Excess Blob Gas Validation**: Calculate excess blob gas from the parent header and ensure it matches the current blocks header parameter excess_blob_gas
+2. **Header Validation**: Compare and validate the current block's header against that of the parent block.
+3. **Ommers Field Check**: Verify that the ommers field in the current block is empty. Note: "ommers" is the gender-neutral term that replaces the previously used term "uncles."
+4. **Block Execution**: Execute the transactions within the block, which yields the following outputs:
+   - **Gas Used**: The total gas consumed by executing all transactions in the block.
+   - **Trie Roots**: The roots of the tries for all transactions and receipts contained in the block.
+   - **Logs Bloom**: A bloom filter of logs from all transactions within the block.
+   - **State**: The state, as specified in the python execution specs, after executing all transactions.
+5. **Header Parameters Verification**: Confirm that the parameters returned from executing the block are present in the block header. This includes comparing the state's root with the `state_root` field in the block header.
+6. **Block Addition**: If all checks are successful, append the block to the blockchain.
+7. **Pruning Old Blocks**: Remove blocks that are older than the most recent 255 blocks from the blockchain.
+8. **Error Handling**: If any validation checks fail, raise an "Invalid Block" error. Otherwise, return None.
+
+Client Code :
+
+| function | EELS(cancun) | Geth | Reth | Erigon | Nethermind | Besu |
```

Comment:
This is pretty cool, I would like to have this table filled for each implementation. However, I think this section more fits in the architecture doc and here I would just focuse on EELS.